bin/cloudtrail_handler.py

```diff
@@ -3,16 +3,17 @@
 It adds new entries to a dynamodb table containing all queries
 """
 
-import boto3
-import json
 import gzip
+import json
 import logging
 from datetime import datetime, timezone
 from io import BytesIO
 
-import settings
-from model import AthenaQuery, QueryState, TIMESTAMP_FORMAT
-from query_dao import QueryDao
+import boto3
```

Comment:
Change to relative imports, so testing from separate directory package is possible.
This will happen across the PR.